frontend/src/components/PageHeading/PageHeading.module.css

```diff
@@ -63,7 +63,7 @@
     text-align: center;
 
     & .title {
-      font: var(--cpd-font-heading-lg-semibold);
+      font: var(--cpd-font-heading-md-semibold);
```

Comment:
Need to check whether this also applies to other places where this is being used